### ISSUE TITLE
SpaceNet和WHU-Mix数据集处理的问题

### ISSUE THREAD

**Issue by wind4jps** [floatingstarZ/OpenRSD#10]
> WHU-Mix [28] is a large-scale building detection dataset containing 43,778 images with spatial resolutions ranging from 0.09 to 2.5 meters. We train on the training and validation sets and evaluate on the test set. SpaceNet [37] is a building detection dataset covering multiple cities. We merge four cities to construct a training set of 10,152 images and evaluate on the merged test set.

请问下是否方便告知SpaceNet用的是哪四个城市的数据，同时训练集和测试集是如何划分的（因为官方下载的SpaceNet的test集没有标注文件）？
以及WHU-Mix数据集是怎么从tif格式的分割数据标注转成检测数据集标注格式的？
